packages/apollo-gateway/src/datasources/RemoteGraphQLDataSource.ts

```diff
@@ -183,9 +185,9 @@ export class RemoteGraphQLDataSource<TContext extends Record<string, any> = Reco
     >,
   ): ValueOrPromise<GraphQLResponse>;
 
-  public didEncounterError(error: Error, _request: Request) {
-    throw error;
```

Comment:
I don't quite understand what the original intention of this `throw` was, but I like the idea of removing it in the default implementation!